askup/views.py

```diff
@@ -735,24 +694,22 @@ def answer_evaluate(request, answer_id, evaluation):
     )
     filter = request.GET.get('filter', None)
     is_quiz_start = request.GET.get('quiz_start', None)
-    answer = get_object_or_404(Answer, pk=answer_id)
+    answer = Answer.objects.filter(pk=answer_id).first()
 
     if not do_user_checks_and_evaluate(request.user, answer, evaluation):
```

Comment:
It seems that qset_id would be a good parameter to use inside `do_user_checks_and_evaluate` function

Reply:
done